src/Components/Users/users.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import { I18Y } from '../../constants';
+
+import style from './users.module.css';
+
+const Users = (props) => {
+  if (props.users.length === 0) {
+    props.setUsers([{
```

Comment:
Consider moving this array of objects to a separate variable outsider of Users.

Reply:
i did it.